src/Webserver.cc

```diff
@@ -121,16 +121,22 @@ bool Webserver::parse_config(const char* file_name) {
 }
 
 RequestHandler* Webserver::get_handler(std::string uri) {
-    size_t last = uri.find_last_of("/");
-    std::string prefix = uri.substr(0, last);
-    std::unordered_map<std::string, RequestHandler*>::const_iterator found = handler_map.find(prefix);
+    std::unordered_map<std::string, RequestHandler*>::const_iterator found = handler_map.find(uri);
```

Comment:
Er, cause you're finding the last /, so if the person is trying to access a subdirectory, it'll consider it as part of the URI prefix. That's why we have to do longest prefix match. For example, if the config is 
path /static StaticFileHandler {
root example
}
and the request is /static/sub/file.txt
Then the prefix should match /static, and look for the file example/sub/file.txt

If you find the last /, then the prefix turns into /static/sub